### PR TITLE
Fix resource pool creation with '/' in name #261

### PR DESCRIPTION
**Pull request by rlazoryshchak**
Now we can create resource pool with '/' in resource pool and root resource pool namespace.

### CODE REVIEW

**Comment by rlazoryshchak** [comment]
@ezr-ondrej and @chris1984 could you take a look?

File: lib/fog/vsphere/requests/compute/create_resource_pool.rb [hunk 1]
```diff
@@ -6,17 +6,19 @@ def create_resource_pool(attributes = {})
           cluster = get_raw_cluster(attributes[:cluster], attributes[:datacenter])
 
           root_resource_pool = if attributes[:root_resource_pool_name]
-                                 cluster.resourcePool.find attributes[:root_resource_pool_name]
+                                 cluster.resourcePool.find attributes[:root_resource_pool_name].gsub('/', '%2f')
```
Comment (ezr-ondrej) [inline lib/fog/vsphere/requests/compute/create_resource_pool.rb:9]: I guess this is the main part of the fix?
Why we do not need this in another resources?
I thought that translating this characters would be a job for the underlying communication library :thinking:

**Comment by ezr-ondrej** [comment]
@rlazoryshchak could you add a comment why do we need the transpolation of `/`, please?
Apart of it :+1:

**Comment by rlazoryshchak** [comment]
This is fix naming of resource pools which contains '/' symbol. Bug coused by rbvmomi transpolation '/' to '%2f' and while creating nested resource pool we can't find it in parent resource pool which contains '/'.

**Comment by ezr-ondrej** [comment]
I've added the comment in the commit comment to be easily found in the future, thanks @rlazoryshchak :+1:

**Pull request merged**